karma.conf.js

```diff
@@ -4,9 +4,6 @@ import { getKarmaConfig } from 'grumbler-scripts/config/karma.conf';
 
 import { TEST } from './webpack.config';
 
-// eslint-disable-next-line no-process-env
-process.env.CHROME_BIN = require('puppeteer').executablePath();
```

Comment:
Hmm weird, thought this was needed to fix something else. but if it passes then sounds good 🤷‍♂️

Reply:
I think it was intended to fix the original problem by using Puppeteer's version of Chromium rather than the system's version, but in this case we have more control over the system's version so it makes sense to remove this line.

Reply:
👍 works for me